_drafts/oryx/presets.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Presets
+description: Presets are used to install predefined applications
+template: concept-topic-template
+last_updated: Apr 4, 2023
+---
+
+# Presets
+
+The [presets package](https://www.npmjs.com/package/@spryker-oryx/presets) contains standard feature sets and resources that can be used to create sample application without writing [boilerplate](./boilerplate.md). While presets might be too opinionated to use for a production application, they are a great way to get started quickly.
+
+Presets are typically used to demonstrate or try out Oryx applications in a fast manner. Product applications will most likely setup the boilerplate in more optimised way, by leaving out features that are not used.
+
+The presets are likely not the perfect production setup for your project, as it comes with a full feature set. In a production setup, you might want to skip some features that included in the preset.
+
+## Dependencies
+
+Presets are provided in a separate [npm package](https://www.npmjs.com/package/@spryker-oryx/presets).
+
+The standard boilerplate uses the presets as the single package to install Oryx applications. The preset application contains dependencies on _all_ [Oryx npm packages](https://www.npmjs.com/org/spryker-oryx). This is done intentionally to ease the (demo) installation. However, for production installation, it is unlikely that you'd need all packages. It could make sense to not use the presets package in your production application. A more fine-grained setup would make more sense.
+
+## Feature sets
```

Comment:
Maybe we should also mention that i18n translations are part of the presets package as well?

Reply:
We've added i18n initially to presets, but I don't think we like to maintain it there:
- we avoid labels in our **code base**
- we maintain a few demo labels in the labs package
- the i18n logic sits inside i18n package